### ISSUE TITLE
valet share broken

### ISSUE THREAD

**Issue by capitaladot** [weprovide/valet-plus#229]
```
valet-project-path $ valet share
Password:
bash: /cli/scripts/fetch-share-url.sh: No such file or directory
sudo: /bin/ngrok: command not found
```

**Comment by capitaladot** [comment]
Probably related to https://github.com/weprovide/valet-plus/issues/216
but it failed slightly differently

**Comment by samgranger** [comment]
Can you check if `~/.composer/vendor/bin` is in your systems path? You can check this by running`echo $PATH` in terminal.

If this is not the case, please execute the following command and restart your terminal: `export PATH=$PATH:~/.composer/vendor/bin`

**Comment by capitaladot** [comment]
Yeah, it is. It is the broken path variable. This works in 7.1 mode.

**Comment by samgranger** [comment]
Can you please check if the fix for #216 solved this issue for you?
